site/content/en/docs/tasks/run/multikueue/job.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Run Kubernetes Job in Multi-Cluster"
+linkTitle: "Kubernetes Job"
+weight: 2
+date: 2024-11-05
+description: >
+  Run a MultiKueue scheduled Kubernetes Job.
+---
+
+## Before you begin
+
+Check the [MultiKueue installation guide](/docs/tasks/manage/setup_multikueue) on how to properly setup MultiKueue clusters.
+
+For the ease of setup and use we recommend using at least Kueue v0.8.1 and Kubernetes v1.30.0.
+
+## Run Kubernetes Job example
+
+Once the setup is complete you can test it by running the [`job-sample.yaml`](/docs/tasks/run/jobs/#1-define-the-job) Job. 
+
+The recommended way of running MultiKueue depends on the configuration of the `JobManagedBy` feature gate in your cluster. 
+
+NOTE: The `JobManagedBy` feature gate is disabled in 1.30 and 1.31 by default, and will be enabled in 1.32 by default.
+
+## Cluster with JobManagedBy enabled
```

Comment:
Could we make this and `Cluster with JobManagedBy disabled` sections sub-section of the `Before you begin`?

Reply:
+1, I would also suggest to move the example "## Run Kubernetes Job example" below